.azure/pipelines/ci-official.yml

```diff
@@ -0,0 +1,226 @@
+trigger:
+  branches:
+    include:
+    - master
+    - release/*
+    exclude:
+    - release/2.0
+
+phases:
+- phase: Windows
+  queue:
+    name: DotNetCore-Windows
+    timeoutInMinutes: 120
+    matrix:
+      Release:
+        BuildConfiguration: Release
+  variables:
+    CI: true
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+    # This variable is required by MicroBuildSigningPlugin to determine permissions for codesigning.
+    TeamName: AspNetCore
+
+    # SignType = { test, real }
+    # This is prefixed underscore because variables automatically become environment variables (and therefore MSBuild properties),
+    # and this one was causing issues in MSBuild projects which use the $(SignType) MSbuild prop.
+    _SignType: real
+
+  steps:
+  - task: NodeTool@0
+    displayName: Install Node 10.x
+    inputs:
+      versionSpec: 10.x
+  - task: MicroBuildSigningPlugin@1
+    displayName: Install MicroBuild plugin
+    condition: and(succeeded(), in(variables['_SignType'], 'test', 'real'))
+    inputs:
+      signType: $(_SignType)
+      zipSources: false
+  # TODO: configure build.cmd to build both x64 and x86 in one invocation
+  # TODO build.cmd -ci
+  - script: build.cmd -ci /p:SkipTests=true /p:Configuration=$(BuildConfiguration) /p:BuildNumber=$(Build.BuildNumber) /t:Build /t:BuildSharedFx /p:SharedFxRID=win-x64 /t:BuildFallbackArchive
+    displayName: Build NuGet packages and win-x64 runtime
+  - script: build.cmd -ci /p:SkipTests=true /p:Configuration=$(BuildConfiguration) /p:BuildNumber=$(Build.BuildNumber) /t:BuildSharedFx /p:SharedFxRID=win-x86
+    displayName: Build win-x86 runtime
+  - powershell: >
+      src/Installers/Windows/clone_and_build_ancm.ps1
+      -GitCredential '$(dn-bot-devdiv-build-rw-code-rw)'
+      -Config $(BuildConfiguration)
+      -BuildNumber $(Build.BuildNumber)
+      -SignType $(_SignType)
+    displayName: Build ANCM installers
+  # TODO: configure harvesting to run as a part of build.cmd
+  - powershell: >
+      src/Installers/Windows/build.ps1
+      -x64 artifacts/runtime/aspnetcore-runtime-internal-2.1.6-servicing-$(Build.BuildNumber)-win-x64.zip
+      -x86 artifacts/runtime/aspnetcore-runtime-internal-2.1.6-servicing-$(Build.BuildNumber)-win-x86.zip
```

Comment:
Are these two supposed to have hard-coded patch versions?

Reply:
It's gross that they're hard coded here. Removing this is what I mean by "TODO: configure harvesting to run as a part of build.cmd"

I definitely consider this VSTS build definition a WIP. It doesn't produce anything official yet. I added it here because it's useful to have _something_ even if incomplete so we can occasionally keep trying VSTS builds to see if progress is being made on BYOC and other features.

Reply:
Fine with me.